rqd/rqd/rqconstants.py

```diff
@@ -69,6 +69,7 @@
 RQD_BECOME_JOB_USER = True
 RQD_CREATE_USER_IF_NOT_EXISTS = True
 RQD_TAGS = ''
+RQD_USE_FQDN = False
```

Comment:
Please add a comment describing the purpose.

Reply:
Added